1주차/1059.py

```diff
@@ -0,0 +1,24 @@
+L = int(input())
+s = list(map(int, input().split()))
+s = sorted(s)
+n = int(input())
+
+answer = []
+cnt = 0
+if n in s:
+    print(0)
```

Comment:
여기는 exit() 사용하면 이후 모든 코드의 들여쓰기를 한칸씩 전부 지울 수 있어요
```suggestion
if n in s:
    print(0)
    exit()
```